mlflow/server/js/src/components/ExperimentRunsTableCompactView.js

```diff
@@ -485,10 +505,11 @@ class ExperimentRunsTableCompactView extends PureComponent {
                   }}
                   style={{...styles.columnStyle, borderLeft: BORDER_STYLE}}
                   cellRenderer={({rowIndex, rowData, parent, dataKey}) => {
-                    // Add extra padding to last row so that we can render dropdowns for bagged
-                    // param key-value pairs in that row
-                    const paddingOpt = rowIndex === rows.length - 1 ? {paddingBottom: 95} : {};
                     const colIdx = NUM_RUN_METADATA_COLS + unbaggedParams.length;
+                    // Add extra padding for load more
+                    const paddingOpt = rowIndex === rows.length - 1
+                      ? { paddingBottom: LOAD_MORE_ROW_HEIGHT * 2 }
```

Comment:
Ideally, this should equal to `FINAL_ROW_HEIGHT + LOAD_MORE_ROW_HEIGHT` with empty cell present in a row. But `FINAL_ROW_HEIGHT` is not available at this stage so I made this 2x the LOAD_MORE_ROW_HEIGHT as a rough accommodation for empty cells.